### ISSUE TITLE
Rationalise Scripts and Run Final Experiments

### ISSUE THREAD

**Issue by j6mes** [sheffieldnlp/fever-baselines#15]
To run
- [ ] MLP: Train on FNC, Evaluate on FNC, Evaluate on FEVER 3 way
- [x] MLP: Train on FEVER with sampled negative pages, Test 
- [x] MLP: Train on FEVER with IR negative pages, Test
- [x] DR: Final score for recall/precision/MRR
- [x] DR: Score using Oracle RTE component
- [ ] RTE: Pre-trained model, evaluate on FEVER
- [x] RTE: Train on FEVER bodies, evaluate on FEVER

Extra:
- BiDAF: Precision/Recall of pretrained model
- BiDAF: FEVER Accuracy using pretrained model on DRQA Pages
- RTE: Train on BiDAF retrieved model: evaluate P/R of BiDAF. Evaluate FEVER score
